src/main/kotlin/org/janelia/saalfeldlab/paintera/meshes/managed/MeshManagerWithSingleMesh.kt

```diff
@@ -89,4 +101,22 @@ class MeshManagerWithSingleMesh<Key>(
         key?.let { createMeshFor(it) }
     }
 
+    private fun MeshGenerator.State.setup() = setupGeneratorState(this)
+
+    @Synchronized
+    private fun setupGeneratorState(state: MeshGenerator.State) {
+        LOG.debug("Setting up state for mesh key {}", meshKey)
+        state.settings.finestScaleLevel = 1
+        state.settings.finestScaleLevel = 0
```

Comment:
One of these should probably be the coarsest scale level? Although, since in the next lines it gets bound to `settings`, it seems that the values will be overwritten by the binding anyway

Reply:
Thanks for catching this @igorpisarev